src/vs/workbench/api/common/extHostTerminalService.ts

```diff
@@ -867,8 +871,9 @@ export abstract class BaseExtHostTerminalService extends Disposable implements I
 	}
 }
 
-class EnvironmentVariableCollection implements vscode.EnvironmentVariableCollection {
+class EnvironmentVariableCollection {
 	readonly map: Map<string, IEnvironmentVariableMutator> = new Map();
+	readonly scopedCollections: Map<string, vscode.EnvironmentVariableCollection & { onDidChangeCollection: Event<void> }> = new Map();
```

Comment:
This should be private, and we can use the `ScopedEnvironmentVariableCollection` type instead